### PR TITLE
feature: add http finctions in separate files and create cmd script to copy fsdata files

### PR DESCRIPTION
**Pull request by TenGosc007**
Add http functions in separate files and create cmd script to copy fsdata files

### CODE REVIEW

File: LWIP/App/lwip.c [hunk 1]
```diff
@@ -64,10 +64,10 @@ void MX_LWIP_Init(void)
...
   NETMASK_ADDRESS[2] = 255;
   NETMASK_ADDRESS[3] = 0;
-  GATEWAY_ADDRESS[0] = 192;
-  GATEWAY_ADDRESS[1] = 168;
-  GATEWAY_ADDRESS[2] = 1;
-  GATEWAY_ADDRESS[3] = 1;
+  GATEWAY_ADDRESS[0] = 0;
+  GATEWAY_ADDRESS[1] = 0;
+  GATEWAY_ADDRESS[2] = 0;
+  GATEWAY_ADDRESS[3] = 0;
```
Comment (Piotrekq) [inline LWIP/App/lwip.c:71]: what?

File: STM32F4_MODBUS_TCP.ioc [hunk 2]
```diff
@@ -4,16 +4,14 @@ ETH.MediaInterface=ETH_MEDIA_INTERFACE_RMII
 File.Version=6
```
Comment (Piotrekq) [inline STM32F4_MODBUS_TCP.ioc:4]: git ignore

File: Middlewares/Third_Party/LwIP/src/apps/http/fsdata_custom.c [hunk 3]
```diff
@@ -26,39 +26,64 @@ static const unsigned char data_404_html[] = {
 	0x79, 0x3E, 0x0A, 0x3C, 0x2F, 0x68, 0x74, 0x6D, 0x6C, 0x3E,
```
Comment (Piotrekq) [inline Middlewares/Third_Party/LwIP/src/apps/http/fsdata_custom.c:26]: git ignore

File: .mxproject [hunk 4]
```diff
@@ -9,34 +9,34 @@ CDefines=USE_HAL_DRIVER;STM32F407xx;USE_HAL_DRIVER;USE_HAL_DRIVER;
 [PreviousGenFiles]
```
Comment (Piotrekq) [inline .mxproject:9]: git ignore

File: fsdata.c [hunk 5]
```diff
@@ -0,0 +1,76 @@
+static const unsigned char data_index_html[] = {
+	/* File: /index.html */
```
Comment (Piotrekq) [inline fsdata.c:2]: git ignore

File: fsdata_custom.c [hunk 6]
```diff
@@ -0,0 +1,89 @@
+static const unsigned char data_404_html[] = {
```
Comment (Piotrekq) [inline fsdata_custom.c:1]: git ignore

File: LWIP/App/lwip.c (hunk 1 above)
Comment (TenGosc007) [reply LWIP/App/lwip.c:71]: Interesting....

File: STM32F4_MODBUS_TCP.ioc (hunk 2 above)
Comment (TenGosc007) [reply STM32F4_MODBUS_TCP.ioc:4]: It have to be updating. This is MX configuration file

**Review by Piotrekq — APPROVED** [review]

**Pull request merged**